### ISSUE TITLE
Most of the sources came back with no result

### ISSUE THREAD

**Issue by Inirakka** [Bionus/imgbrd-grabber#3302]
**Bug description**

As the title said, most of the sources came back with no result, no matter what i do, it keeps happening, even trying different version still do the same.

**Context**

[main_2024.09.22-09.55.44.876.log](https://github.com/user-attachments/files/17088081/main_2024.09.22-09.55.44.876.log)
[settings..txt](https://github.com/user-attachments/files/17088082/settings.txt)

**Screenshots**

![Screenshot 2024-09-22 100018](https://github.com/user-attachments/assets/7af63fab-8469-45ea-81cb-77ae6afe46c9)
Some source like kemono do show result, but the other just didn't
![Screenshot 2024-09-22 095609](https://github.com/user-attachments/assets/3a10573a-d8c8-46ae-a82a-7bbbac905efd)

**System information**

- OS: Windows 11
- Grabber version: 7.12.12 (other version have the same bug on my end, the nightly too)

**Additional context**

**Comment by Bionus** [comment]
Hello 👋 

I see a bunch of parsing errors, which leads me to think what the network returns is not what Grabber expects.
Also, with the fact that the last page is "internetpositif.id", could it be that your IP has been blocked somehow? Or that your network provider / country (Indonesia?) is blocking those domains?

Does it work when using a VPN?

**Comment by Inirakka** [comment]
Yes most of these sites is blocked in my country but it used to be fine when i use vpn or warp on the grabber, now nothing is working, also android version came back with the same result too

**Comment by Inirakka** [comment]
Also, when it works i backup the setting folder content, and pasting it to the directory before opening the app make it work too
